src/ExtEventLoop.php

```diff
@@ -178,8 +178,7 @@ public function tick()
 
         $this->futureTickQueue->tick();
 
-        // @-suppression: https://github.com/reactphp/react/pull/234#discussion-diff-7759616R226
-        @$this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
+        $this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
```

Comment:
@jmalloc Thanks for the feedback!

> IIRC, the initial problem was that `EventBase::loop()` emits an (additional?) warning after an error in a user-defined callback. So, perhaps calling `trigger_error()` in a timer function, or something like that would reproduce it. I can't recall if I witnessed this myself, but I kept the @-suppression at @nrk's request.

Yes, i thought so, but couldn't reproduce it. Maybe it's fixed in the extension itself in the meantime.

I'd say it's *better* to have additional warnings than suppressing everything. Any thoughts @reactphp/core?